smart-contracts/assembly/contracts/FT/__tests__/ft-burn.spec.ts

```diff
@@ -92,3 +109,61 @@ describe('Fails burn ERC20', () => {
     burn(new Args().add(u256.Max).serialize()),
   );
 });
+
+const allowAmount = new u256(1, 1, 1, 1);
+describe('burnFrom', () => {
+  beforeAll(() => {
+    switchUser(user3Address);
+
+    // Increase allowance for U1 to spend U3 tokens
+    increaseAllowance(
+      new Args().add(user1Address).add(allowAmount).serialize(),
+    );
+    switchUser(user1Address);
+  });
+
+  throws('Fails because not enough allowance U3 => U1 ', () => {
```

Comment:
I'm not sure that the U3 and U1 concept will be simple to understand for the dev in charge of fixing the UT if they fail.

```suggestion
  throws('on insufficient allowance', () => {
```